frontend/src/pages/notebookController/screens/server/StopServerModal.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { Button, Modal, ModalVariant } from '@patternfly/react-core';
+import { Notebook } from '../../../../types';
+import { deleteNotebook } from '../../../../services/notebookService';
+import useNamespaces from '../../useNamespaces';
+import useNotification from '../../../../utilities/useNotification';
+import { allSettledPromises } from '../../../../utilities/allSettledPromises';
+
+type StopServerModalProps = {
+  notebooksToStop: Notebook[];
+  onNotebooksStop: (stop: boolean) => void;
```

Comment:
Nit, but really tiny... variable naming is hard 🙂 
```suggestion
  onNotebooksStop: (didStop: boolean) => void;
```